simcon_project/users/views/researcher_home.py

```diff
@@ -1,7 +1,22 @@
 from django.shortcuts import render
+from django.db.models import Q
 from conversation_templates.models import TemplateResponse
 from django.contrib.auth.decorators import user_passes_test
-from django.core.paginator import Paginator
+import django_tables2 as tables
+from django_tables2 import RequestConfig
+
+
+class ResponseTable(tables.Table):
+    name = tables.columns.Column(accessor="student.get_full_name")
```

Comment:
We need an order_by="student.last_name" attribute here so since clicking on the column header to sort causes an error right now. Might be good to change get_full_name in custom-user to change the name format to last name, first name.

Reply:
You could also build the string in researcher_view and pass it in as a dict. Then the accessor is the key you pick